From e6432a827df17c3ffe9438647ee6f75f180fe122 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Mon, 12 Jul 2010 10:20:27 +0800
Subject: [PATCH] ENGR00125042 mx35: Fix error in a comment for header file

Fix error in a comment for header file
Fix ppp review about error board description

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 include/configs/mx35_3stack_mfg.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/configs/mx35_3stack_mfg.h b/include/configs/mx35_3stack_mfg.h
index c056521..ba962ef 100644
--- a/include/configs/mx35_3stack_mfg.h
+++ b/include/configs/mx35_3stack_mfg.h
@@ -3,7 +3,7 @@
  *
  * (C) Copyright 2008-2010 Freescale Semiconductor, Inc.
  *
- * Configuration settings for the MX31ADS Freescale board.
+ * Configuration settings for the MX35 3stack Freescale board.
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License as
-- 
1.5.4.4

